packages/jsii-diff/generate.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
why do we need this again?

Reply:
Copied from jsii to embed a build-time version string.